Monitor pull requests and issues per project

itsikm opened the pull request.

Commit:
- Add config.js & organize the project

Commit:
Organizing the project:
- Add prod.json as default config
- Add Github token to configuration
- Using logger instead of console

Commit:
Monitor pull requests and issues per project:
- Add Github class
- Update Github job

Commit:
Monitor pull requests and issues per project:
- Add Github class
- Update Github job
- Fix small bugs

guy-mograbi-at-gigaspaces commented on jobs/github.job.js:
```diff
@@ -1,27 +1,285 @@
-        send_event('github', { current: currentRequests, last: lastRequests });
+    },
+    {
+        user: 'guy-mograbi-at-gigaspaces',
```
Can't we construct this at runtime? This was the whole idea.

Commit:
Monitor pull requests and issues per project:
- Update Github class to pull repos automatically
- Update Github job to collect issues from repos data
- Add exclude repos option

guy-mograbi-at-gigaspaces commented on jobs/jira-issues.job.js:
```diff
@@ -40,49 +43,29 @@ function addUserIssue(issue) {
-    catch(e) {
-        console.log('addUserIssue:error', e, issue);
+    catch (e) {
+        logger.info('addUserIssue:error', e, issue);
```
logger.error?

Commit:
Monitor pull requests and issues per project:
- Replace logger.info with logger.error
